### PR TITLE
Update ATAC wdl to include all tasks

### PR DESCRIPTION
**Pull request by aawdeh**
Updated ATAC WDL to include barcode correction and splitting the fastq steps. 

SplitFastq splits the fastq files into small fastq files. The scatter step in the WDL (similar to Optimus) is added to run the TrimAdapters/BWA tasks simultaneously on the multiple smaller fastq files.  

Update the AddCBtags task to add both the CR and CB tags to the output BAM file. In the BAM file, so far, all reads with valid and invalid barcodes are included. Reads with valid barcodes have both the CB and CR tags, while reads with invalid barcodes only have the CR tags. Updated bash command to reflect that. (This may need to be updated again if only reads with valid barcodes are required.)

We ran this WDL in Terra. The job run can be found here: https://app.terra.bio/#workspaces/warp-pipelines/atac-multiome-bwa/job_history/2b487c5a-6acb-44a7-b391-9c47df8fbe73 
### Checklist 
If you can answer "yes" to the following items, please add a checkmark next to the appropriate checklist item(s) **and** notify our WARP documentation team by tagging either @ekiernan or @kayleemathews in a comment on this PR.

- [x] Did you add inputs, outputs, or tasks to a workflow?
- [x] Did you modify, delete or move: file paths, file names, input names, output names, or task names?
- [ ] If you made a changelog update, did you update the pipeline version number?

### CODE REVIEW

**Comment by nikellepetrillo** [comment]
retest this please

**Review by ekiernan — APPROVED** [review]
Thanks, Aseel!

File: beta-pipelines/multiome/atac.wdl [hunk 1]
```diff
@@ -1,5 +1,8 @@
 version 1.0
 
+import "../../tasks/skylab/MergeSortBam.wdl" as Merge
+#import "../../../tasks/skylab/FastqProcessing.wdl" as FastqProcessing
```
Comment (nikellepetrillo) [inline beta-pipelines/multiome/atac.wdl:4]:
```suggestion
```

File: beta-pipelines/multiome/atac.wdl [hunk 2]
```diff
@@ -43,125 +49,170 @@ workflow ATAC {
...
+        cpu: "(optional) the number of cpus to provision for this task"
+        disk_size: "(optional) the amount of disk space (GiB) to provision for this task"
+        preemptible: "(optional) if non-zero, request a pre-emptible instance and allow for this number of preemptions before running the task on a non preemptible machine"
+   }
+
+  command <<<
+  
+    set -euo pipefail
+    
+    # Cat files for each r1, r3 and barcodes together [? is this a good idea or should we just do what optimus does]
```
Comment (nikellepetrillo) [inline beta-pipelines/multiome/atac.wdl:159]: "[? is this a good idea or should we just do what optimus does]" is this still an open question?

Comment (aawdeh) [reply beta-pipelines/multiome/atac.wdl:159]: I just added this as a comment for us for later improvements we might want to do. We can definitely remove the comment and just keep note of it for later on.

**Review by nikellepetrillo — APPROVED** [review]

**Pull request merged**